lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,11 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            message = "This line has {{numberOfStatementsOnThisLine}} statement(s). Maximum allowed is {{maxStatementsPerLine}}.";
```

Comment:
That's a good point, @ljharb. IIRC, the position in the past has been that those wanting to do computer consumption of the output should use a formatter intended for non-human consumption or use the API directly. Does that sound reasonable?